source/common/common/shared_memory_hash_set.h

```diff
@@ -317,11 +318,17 @@ template <class Value> class SharedMemoryHashSet : public Logger::Loggable<Logge
   };
 
   // It seems like this is an obvious constexpr, but it won't compile as one.
-  static size_t alignment() {
+  static size_t calculateAlignment() {
     return std::max(alignof(Cell), std::max(alignof(uint32_t), alignof(Control)));
   }
 
-  static uint32_t align(uint32_t size) { return (size + alignment() - 1) & ~(alignment() - 1); }
+  static uint32_t align(uint32_t size) {
+    const size_t alignment = calculateAlignment();
```

Comment:
I just remembered that there's already an implementation of this in stats_impl.cc at the top: roundUpMultipleNaturalAlignment().  Can you put it in a common location and share the function between the two?  Or add a TODO?